scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1262,7 +1405,17 @@ public String getTaskServerLogsByTag(String jobId, String taskTag)
     @ImmediateService
     public List<SchedulerUserInfo> getUsersWithJobs() throws NotConnectedException, PermissionException {
         frontendState.checkPermission("getUsersWithJobs", "You don't have permissions to get users with jobs");
-        return dbManager.loadUsersWithJobs();
+        synchronized (waitForJobsToBeSubmitted) {
+            while (!waitForJobsToBeSubmitted.isEmpty()) {
+                try {
+                    waitForJobsToBeSubmitted.wait();
+                } catch (InterruptedException e) {
+                    logger.warn("Thread was interrupted.", e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)